examples/ssr/package.json

```diff
@@ -16,6 +19,13 @@
     "react-dom": "17.0.2"
   },
   "devDependencies": {
-    "razzle": "^3.4.5"
+    "babel-preset-razzle": "^4.2.14",
```

Comment:
as per [Razzle upgrade guide from 3.x to 4.x](https://razzlejs.org/docs/upgrade-guide)